### ISSUE TITLE
Set up Google analytics for refinebio-examples pages

### ISSUE THREAD

**Issue by cansavvy** [AlexsLemonade/refinebio-examples#281]
I don't know how to do this, but its something @dvenprasad recommended. We want to know what people are clicking on and using the most. Feedback is helpful!

This seems like it should help: 
https://support.google.com/analytics/answer/1008015?hl=en

**Comment by jashapiro** [comment]
We will probably want to make use of the include options as I described [here](https://github.com/AlexsLemonade/refinebio-examples/issues/279#issuecomment-706194235) to get this into the header.

**Comment by cansavvy** [comment]
Okay. I tried to do this and it doesn't seem to be working. There's so many buttons and I'm quite lost: https://analytics.google.com/analytics/web/?authuser=1#/p249911149/realtime/overview?params=_u..nav%3Ddefault

**Comment by jashapiro** [comment]
What is that link supposed to be taking us to? What did you try? What are your questions?

**Comment by cansavvy** [comment]
> What is that link supposed to be taking us to? What did you try? What are your questions?

I had so many questions but it turned out I was doing it all wrong anyway, @dvenprasad says we set this up through ALSF's already standing Google Analytics account which I didn't realize was a thing.

**Comment by cansavvy** [comment]
But a question that is still pertinent is will https://github.com/AlexsLemonade/refinebio/issues/2542 affect the domain name we provide for this?

**Comment by jashapiro** [comment]
> But a question that is still pertinent is will [AlexsLemonade/refinebio#2542](https://github.com/AlexsLemonade/refinebio/issues/2542) affect the domain name we provide for tracking?

I would go with the assumption that we will be able to change it later if we need to.

**Comment by cansavvy** [comment]
Okay. I've coordinated with Craig and we need to add this a Google Analytics script with tags and etc at the top of all our htmls. I'll file a PR with this.

**Comment by cansavvy** [comment]
Wanted to check with Craig before I posted the blurb publicly. This is what needs to be posted at the top of each html. 
```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-C59ZQK86CL"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-C59ZQK86CL');
</script>
```

**Comment by jashapiro** [comment]
You will likely want to add this not just at the top, but inside the `<head>` tag, specifically. I think you can do that with `rmarkdown::includes(in_header =  'google_analytics.html')` similar to what you did in #307

**Comment by cansavvy** [comment]
> You will likely want to add this not just at the top, but inside the `<head>` tag, specifically. I think you can do that with `rmarkdown::includes(in_header = 'google_analytics.html')` similar to what you did in #307

Ah this is much less janky of a solution than what I was just working on. I forgot about ` `rmarkdown::includes(in_header = )`.

**Comment by jaclyn-taroni** [comment]
Closed by #314 ?
